libs/shared/nestjs/README.md

```diff
@@ -0,0 +1,9 @@
+<!-- gitbook-ignore -->
```

Comment:
There is already a `libs/nest` that I believe have the same purpose as the one you are creating here

Reply:
Ahh nice, in my defence it was added after I created this 😅 .
I'll move this to that library.